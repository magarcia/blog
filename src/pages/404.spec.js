import React from 'react';
import renderer from 'react-test-renderer';
import ErrorPage from './404';

it('renders correctly', () => {
  const props = {
    location: {
      pathname: `/`,
    },
    title: 'title',
  };
  const tree = renderer.create(<ErrorPage {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
