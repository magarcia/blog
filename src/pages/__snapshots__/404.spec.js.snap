// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <header
    className="header"
  >
    <h1>
      <a
        href="/"
      >
        magarcia
      </a>
    </h1>
    <div
      className="darkLightMode "
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
    >
      <svg
        className="feather feather-sun"
        height={24}
        viewBox="0 0 24 24"
        width={24}
        xmlns="http://www.w3.org/2000/svg"
      >
        <title />
        <g
          fill="none"
          strokeLinecap="round"
          strokeWidth="2"
        >
          <circle
            cx="12"
            cy="12"
            r="5"
          />
          <path
            d="M12 1v2"
          />
          <path
            d="M12 21v2"
          />
          <path
            d="M4.22 4.22l1.42 1.42"
          />
          <path
            d="M18.36 18.36l1.42 1.42"
          />
          <path
            d="M1 12h2"
          />
          <path
            d="M21 12h2"
          />
          <path
            d="M4.22 19.78l1.42-1.42"
          />
          <path
            d="M18.36 5.64l1.42-1.42"
          />
        </g>
      </svg>
      <input
        aria-label="Switch between Dark and Light mode"
        checked={false}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="checkbox"
      />
    </div>
  </header>,
  <img
    alt="Lost"
    src="test-file-stub"
    style={
      Object {
        "position": "absolute",
        "width": "45rem",
      }
    }
  />,
  <h1
    style={
      Object {
        "fontSize": "23rem",
      }
    }
  >
    404
  </h1>,
]
`;
